packages/combo-box/test/basic.test.js

```diff
@@ -143,17 +136,6 @@ describe('Properties', () => {
     });
   });
 
-  describe('label property', () => {
-    it('should have undefined by default', () => {
-      expect(comboBox.label).to.be.undefined;
-    });
```

Comment:
The `label` property comes from `LabelMixin` and is tested there:

https://github.com/vaadin/web-components/blob/2c94af2025e4f8c14e2200bb655ed6a031d8abda/packages/field-base/test/label-mixin.test.js#L33-L36